test/unit/Install-OSPlatformServiceCenter.Tests.ps1

```diff
@@ -184,7 +184,7 @@ InModuleScope -ModuleName OutSystems.SetupTools {
             Install-OSPlatformServiceCenter | Out-Null
 
             It 'Should run the RunSCInstaller with specific parameters' {
-                $assParams = @{ 'CommandName' = 'RunSCInstaller'; 'Times' = 1; 'Exactly' = $true;'Scope' = 'Context'; 'ParameterFilter' = { $Arguments -eq "-file ServiceCenter.oml -extension OMLProcessor.xif IntegrationStudio.xif PlatformLogs.xif" }}
+                $assParams = @{ 'CommandName' = 'RunSCInstaller'; 'Times' = 1; 'Exactly' = $true;'Scope' = 'Context'; 'ParameterFilter' = { $Arguments -eq "-file ServiceCenter.oml -extension OMLProcessor.xif IntegrationStudio.xif PlatformLogs.xif CentralizedPlatformLogs.xif" }}
```

Comment:
You are missing a new test case for your changes. Please create one for the specific version you are checking in the code (you need to leave this change for this test case because it is now required as well for 12.X)